### DISCUSSION TITLE
Method of changing the logo without swapping out files in the active codebase

### DISCUSSION THREAD

**Discussion by KyleMaas** [mediacms-io/mediacms#632] [Feature Requests]
**Describe the feature you'd like**
It would be awfully nice if there was a way to change out the logo in the top left corner without just replacing the files in the codebase.  There are a few reasons for this:

* If the codebase can be separated from local customizations, it makes it easier to distribute MediaCMS in an upgradable way using system packages - like installing to `/usr/share` instead of to `/home/mediacms.io`
* It makes it easier to replicate the server, both for upgrade and redundancy purposes, if the code can be kept in a clean upstream state.
* Fewer divergences from git upstream sources for MediaCMS updates

**Describe alternatives you've considered**
I can think of a couple ways to accomplish this:

* Ideally for user-friendliness but more complicated, have a method in the web UI to edit the logo URL locations.
* Add a variable in `settings.py` and `local_settings.py` for the dark and light logo URLs and use these throughout

**Comment by KyleMaas** [comment]
Just ran across the logo URLs in `templates/config/installation/site.html` - is that the more correct way to change those URLs? 
 And if so, could it maybe integrate variables from `settings.py` so the changes are only being made in one configuration file instead of throughout the codebase?

**Comment by mgogoulos** [comment]
Nice suggestions

I have been thinking of providing all the settings to an admin page. So all settings take effect through it, and you don't have to edit python/config files and also restart services. Maybe logo upload could find it's way in it?

**Reply by KyleMaas** [reply]
An admin panel would be very helpful for a program like MediaCMS which is otherwise very graphically configured.

**Comment by alfred-stokespace** [comment]
For some reason my SVG's didn't work, I had to delete the svg references in `templates/config/installation/site.html` and hijack the png files. 

```
diff --git a/templates/config/installation/site.html b/templates/config/installation/site.html
index 80cbf8c..d87098a 100644
--- a/templates/config/installation/site.html
+++ b/templates/config/installation/site.html
@@ -12,11 +12,9 @@ MediaCMS.site = {
     logo:{
        lightMode:{
             img: "/static/images/logo_dark.png",
-            svg: "/static/images/logo_dark.svg",
        },
        darkMode:{
             img: "/static/images/logo_light.png",
-            svg: "/static/images/logo_light.svg",
        },
     },
     pages: {
```
